src/ILLink.RoslynAnalyzer/RequiresISymbolExtensions.cs

```diff
@@ -43,5 +64,22 @@ private static bool IsInRequiresScope (this ISymbol member, string requiresAttri
 
 			return false;
 		}
+
+		/// <summary>
+		/// This method determines if the member has a Requires attribute and returns it in the variable requiresAttribute.
+		/// </summary>
+		/// <param name="member">Symbol of the member to search attribute.</param>
+		/// <param name="requiresAttribute">Output variable in case of matching Requires attribute.</param>
+		/// <returns>True if the member contains a Requires attribute; otherwise, returns false.</returns>
+		private static bool TryGetRequiresAttribute (ISymbol member, string requiresAttribute, [NotNullWhen (returnValue: true)] out AttributeData? requiresAttributeData)
```

Comment:
I think it would be the same here to put:
```C# 
private static bool TryGetRequiresAttribute (ISymbol member, string requiresAttribute, [NotNullWhen (returnValue: true)] out AttributeData? requiresAttributeData) => member.TryGetAttribute(requiresAttribute, requiresAttributeData);
```
or since it's private, we could even just replace the references to `TryGetRequiresAttribute` with `TryGetAttribute`.

Reply:
I enabled the automerge and got merged but I will create another PR for this